src/WebJobs.Script/Config/RouteHandlingOptions.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Script.Configuration
+{
+    public class RouteHandlingOptions
+    {
+        private string _authenticationLevel;
+
+        /// <summary>
+        /// Mode determining how routes are mapped. "function" (default) maps httpTrigger functions
+        /// defined in function metadata. "all" creates a single catch-all route that handles all
+        /// requests and proxies it to the custom handler.
+        /// </summary>
+        public string Mode { get; set; } = "function";
+
+        /// <summary>
+        /// Only applicable to mode = "all". Determines the authentication level for the catch-all route.
+        /// Allowed values: "function", "anonymous".
```

Comment:
> /// Allowed values: "function", "anonymous".

The comment limits to two possible allowed values, but default logic returns `string.Empty` when `function`